### ISSUE TITLE
login page is not working

### ISSUE THREAD

**Issue by abundzugamer** [LibreTime/libretime#894]
zendphp.log:
```
2019-10-17T12:32:49+02:00 ERR (3): xxx [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: ErrorException: Undefined index: id in /usr/share/airtime/php/airtime_mvc/application/controllers/$
Stack trace:
#0 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(308): exception_error_handler(8, 'Undefined index...', '/usr/share/airt...', 308, Array)
#1 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Action.php(516): ApiController->liveInfoAction()
#2 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('liveInfoAction')
#3 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#4 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#5 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#6 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#7 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#8 {main}
2019-10-17T12:32:53+02:00 ERR (3): xxx [ErrorController.php:26 - errorAction()] - Undefined index: id
2019-10-17T12:32:53+02:00 ERR (3): xxx [ErrorController.php:27 - errorAction()] - #0 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(308): exception_error_handler(8, 'Undefined index...', '/usr/sh$
#1 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Action.php(516): ApiController->liveInfoAction()
#2 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('liveInfoAction')
#3 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#4 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#5 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#6 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#7 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#8 {main}
2019-10-17T12:32:53+02:00 ERR (3): xxx [ErrorController.php:54 - errorAction()] - An internal application error has occurred.: ErrorException: Undefined index: id in /usr/share/airtime/php/airtime_mvc/application/controllers/$
Stack trace:
#0 /usr/share/airtime/php/airtime_mvc/application/controllers/ApiController.php(308): exception_error_handler(8, 'Undefined index...', '/usr/share/airt...', 308, Array)
#1 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Action.php(516): ApiController->liveInfoAction()
#2 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('liveInfoAction')
#3 /usr/share/airtime/php/vendor/zf1s/zend-controller/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#4 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#5 /usr/share/airtime/php/vendor/zf1s/zend-application/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#6 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#7 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#8 {main}
```

I blurred the IP address for security reasons

The start page works, but no loginpage.

**Comment by Robbt** [comment]
Please provide more context, ie what release or latest commit if using git and what distro you installed this on. 

We will try to reproduce this and look into your report.

**Comment by abundzugamer** [comment]
I used the latest version of Git
git clone https://github.com/LibreTime/libretime.git

The whole thing runs on ubuntu 16.04

**Comment by Robbt** [comment]
Ok, this looks like it happened due to a recent commit - `$nextID = $result["previous"]["metadata"]["id"];` from #45dbf84750 airtime_mvc/application/controllers/ApiController.php (Codenift             2019-09-19 20:07:50 -0400  308)  see https://github.com/LibreTime/libretime/commit/45dbf8475060e14a56dae4d9389ff8546a69bc09

Will need to look into this more.

**Comment by paddatrapper** [comment]
Is this not fixed in #893?

**Comment by Robbt** [comment]
Ok, cool looks like that we can mark this closed.
